packages/session-replay-browser/src/events/events-idb-store.ts

```diff
@@ -102,16 +103,33 @@ export class SessionReplayEventsIDBStore implements AmplitudeSessionReplayEvents
   loggerProvider: ILogger;
   storageKey = '';
   maxPersistedEventsSize = MAX_EVENT_LIST_SIZE_IN_BYTES;
-  interval = MIN_INTERVAL;
+  interval: number;
   timeAtLastSplit: number | null = null;
 
-  constructor({ loggerProvider, apiKey }: { loggerProvider: ILogger; apiKey: string }) {
+  private readonly minInterval: number;
+  private readonly maxInterval: number;
+
+  constructor({
+    loggerProvider,
+    apiKey,
+    minInterval,
+    maxInterval,
+  }: {
+    loggerProvider: ILogger;
+    apiKey: string;
+    minInterval?: number;
+    maxInterval?: number;
+  }) {
     this.loggerProvider = loggerProvider;
     this.apiKey = apiKey;
+    this.maxInterval = maxInterval ?? MAX_INTERVAL;
+    this.minInterval = minInterval ?? MIN_INTERVAL;
+    this.interval = 0;
   }
 
-  async initialize(sessionId?: number) {
-    const dbName = `${this.apiKey.substring(0, 10)}_amp_session_replay_events`;
+  async initialize(type: EventType, sessionId?: number) {
+    const dbSuffix = type === 'rrweb' ? '' : `_${type}`;
+    const dbName = `${this.apiKey.substring(0, 10)}_amp_session_replay_events${dbSuffix}`;
```

Comment:
Ah I just realized, the change to add `rrweb` will break existing stores on end user's version update, and leave a dangling version of the old `events` db. Instead of using suffix for each type, what about just leaving the existing events store as is, and adding the suffix for click events? Otherwise we'll need to add in a migration here

Reply:
Wow, rereading and I see that's what you have here already! Sorry, misread it as appending `rrweb` the first time I looked at this